Application/__init__.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask
+from .models import db
+
+
+def create_app():
+    app = Flask(__name__)
+
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///' + 'Db/db.sqlite'
+
+    db.init_app(app)
+
+    # to create db
```

Comment:
these comments are hard to find and i did not get it to work when uncommenting them. IT should be either defined in a README howo to ccreate db or using an ENV variable maybe